src/Resources/views/CRUD/base_edit_form_macro.html.twig

```diff
@@ -8,20 +8,35 @@
             <div class="{{ form_group.box_class }}">
                 <div class="box-header">
                     <h4 class="box-title">
-                        {{ form_group.label|trans({}, form_group.translation_domain ?: admin.translationDomain) }}
+                        {% if form_group.translation_domain is same as(false) %}
+                            {{ form_group.label }}
+                        {% else %}
+                            {% set translationDomain = form_group.translation_domain|default(admin.translationDomain) %}
+                            {{ form_group.label|trans({}, translationDomain) }}
+                        {% endif %}
                     </h4>
                 </div>
                 <div class="box-body">
                     <div class="sonata-ba-collapsed-fields">
                         {% if form_group.description %}
-                            <p>{{ form_group.description|trans({}, form_group.translation_domain ?: admin.translationDomain)|raw }}</p>
+                            {% if form_group.translation_domain is same as(false) %}
+                                <p>{{ show_group.description|raw }}</p>
+                            {% else %}
+                                {% set translationDomain = form_group.translation_domain|default(admin.translationDomain) %}
+                                {{ form_group.description|trans({}, translationDomain)|raw }}
```

Comment:
Generally we are using snake case for variables defined in templates.
BTW, I'd like to avoid one time variables.
```suggestion
                                {% set translation_domain = form_group.translation_domain|default(admin.translationDomain) %}
                                {{ form_group.description|trans({}, translation_domain)|raw }}
```

Reply:
I removed the one time variables.